plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -286,4 +286,93 @@ else if (!constraint.isAll()) {
         columns.ifPresent(value -> builder.append(" columns=").append(value));
         return builder.toString();
     }
+
+    public static Builder builder()
+    {
+        return new Builder();
+    }
+
+    public static Builder builder(JdbcTableHandle handle)
```

Comment:
The builder is nice from setComment perspective, but other operations are not useful, if the calling code has only partial information about what the table handle represents.
I.e. in almost all cases, the calling code **may** require update if the table handle gains a new field, so we should **not** have a `builder(JdbcTableHandle)` method.